### PR TITLE
Dask container recipe

### PR DESCRIPTION
**Pull request by cgoina**
# Submitting a Container
 
 (If you're requesting for a new container, please check the procedure described [here](https://github.com/BioContainers/containers#241-how-to-request-a-container).  
 
 ## Check BioContainers' Dockerfile [specifications](https://github.com/BioContainers/specs)  
 ## Checklist
 
 1. Misc
 - [ ] My tool doesn't exist in [BioConda](#make-sure-your-tool-isnt-already-present-in-bioconda)   
 - [ ] The image can be built   
 
 2. [Metadata](#check-biocontainers-dockerfile-metadata)  
 - [ ] LABEL base_image  
 - [ ] LABEL version
 - [ ] LABEL software.version  
 - [ ] LABEL about.summary  
 - [ ] LABEL about.home  
 - [ ] LABEL about.license   
 - [ ] MAINTAINER  
 
 3. Extra (optionals)
  - [ ] I have written tests in test-cmds.txt  
  - [ ] LABEL extra.identifier    
  - [ ] LABEL about.documentation   
  - [ ] LABEL about.license_file
  - [ ] LABEL about.tags   
 
 
 ## Check [BioContainers'](https://github.com/BioContainers/specs) Dockerfile metadata

### CODE REVIEW

**Comment by cgoina** [comment]
This container contains scripts that we use in our nextflow dask workflow, which we intend to submit to nf-core.

File: dask/2023.10.1-py11-ol9/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+RUN mamba env update -n base -f /tmp/conda-env.yml
+
+RUN rm -rf /opt/mambaforge/pkgs
+
+WORKDIR /opt/scripts/daskscripts
+
+ENV DASK_CONFIG=/opt/scripts/daskscripts/config/dask-config.yml
+
+# Add scripts
+COPY --chmod=755 scripts/* /opt/scripts/daskscripts
```
Comment (mboudet) [inline dask/2023.10.1-py11-ol9/Dockerfile]: Seems like the --chmod option (buildkit) is not supported by docker-py, which we use for building the image.

Can you try to add a separate 'chmod' step, to see if we can build it?

**Comment by cgoina** [comment]
I made the change to chmod in a separate step

**Comment by cgoina** [comment]
I saw the failure "Found some errors: software.version label not matching directory version name" - I named the folder based on the dask version, python version and os version - should I include all these in the software.version label? The current software version label is only showing the dask version

**Comment by cgoina** [comment]
I updated the software.version label to reflect the dir name

**Comment by mboudet** [comment]
Ok, you should just remove the extra.identifiers.biotools="dask" , since I don't think it exists in biotools ?

**Comment by mboudet** [comment]
Seems fine to me, merging.

**Pull request merged**